Tests/Bootstrap/DiagnosticLogger.spec.ts

```diff
@@ -0,0 +1,11 @@
+import assert = require("assert");
+import { DiagnosticLogger } from "../../Bootstrap/DiagnosticLogger";
+
+describe("DiagnosticLogger", () => {
+    describe("#DiagnosticLogger.DefaultEnvelope", () => {
+        it("should have the correct version string", () => {
+            const version = require("../../../package.json").version;
+            assert.equal(DiagnosticLogger.DefaultEnvelope.properties.sdkVersion, version);
+        });
+    });
+});
```

Comment:
Does it necessary to add tests for other fields in properties bag? Looks like all of the fields are required.

Reply:
:+1: added test for rest of the properties bag